azure_pipelines/pipeline-trigger.yml

```diff
@@ -0,0 +1,88 @@
+variables:
+  - name: 'releaseVersion'
+    value: 'testVersion'
+  - name: 'commonCoreReleaseVersion'
+    value: null
+  - name: 'frameworkChecksum'
+    value: null
+  - name: 'repositoryName' # Name of the repository
+    value: 'AzureAD/microsoft-authentication-library-for-objc'
+  - name: 'repositoryBranch' # Name of the branch where version number will be updated for files that contain version number.
+    value: 'ameya-master'
+  - name: 'GithubServiceConnection' # Service connection name used to connect Github repository
+    value: 'MSAL ObjC Service Connection'
+
+trigger:
+  branches:
+    include:
+    - ameya-master
```

Comment:
Will this be master branch eventually?

Reply:
No, this is just for testing. I'll both branches when my testing is done